tests/aws/services/lambda_/functions/common/echo/dotnet6/Makefile

```diff
@@ -1,9 +1,20 @@
+UNAME := $(shell uname -m)
+ifeq ($(UNAME),x86_64)
+	ARCHITECTURE ?= x86_64
+else
+	ARCHITECTURE ?= arm64
+endif
+DOCKER_PLATFORM ?= linux/$(ARCHITECTURE)
+# The packaging function architecture is x86_64 by default and needs to be set explicitly for arm64
+# https://github.com/aws/aws-extensions-for-dotnet-cli/blob/cdd490450e0407139d49248d94a4a899367e84df/src/Amazon.Lambda.Tools/LambdaDefinedCommandOptions.cs#L111
+FUNCTION_ARCHITECTURE ?= $(ARCHITECTURE)
```

Comment:
Dotnet is a good example why an explicit `ARCHITECTURE` flag is needed:
* In general, the `--platform=$(DOCKER_PLATFORM)` flag is needed for cross-architecture on multi-architecture platforms
* The Dotnet build specifically requires the `--function-architecture` flag for ARM build. It does not detect the right architecture automatically (unlike Golang builds). Therefore, being consistently verbose clarifies the build options.